src/sass/base/_global.sass

```
@@ -1,16 +1,81 @@
 @import "../base/variables"
+@import "../base/mixins"
 
 html
     font-size: 13px
 
-    body 
-        font-family: $primary-font
-        font-weight: $fw-regular
-        font-style: normal
-        color: $text-color
-
-        a
-            color: $text-color
-            &:hover
-                color: $hover-color
-                text-decoration: none
+body
+    font-family: $primary-font
+    font-weight: $fw-regular
+    font-style: normal
+    color: $text-color
+
+a
+    color: $text-color
+
+    &:hover
+        color: $hover-color
+        text-decoration: none
+
+.transition
+    transition: all .3s ease-in-out
+    -moz-transition: all .3s ease-in-out
+    -webkit-transition: all .3s ease-in-out
+    -o-transition: all .3s ease-in-out
+    -ms-transition: all .3s ease-in-out
+
+.headingTitle
```

```suggestion
.heading-title
```
class đặt theo dạng snake thay vì camelCase nhé